config/samples/policies_v1alpha1_clusteradmissionpolicy.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: policies.kubewarden.io/v1alpha1
 kind: ClusterAdmissionPolicy
 metadata:
-  name: psp-capabilities
+  name: capabilities-psp
 spec:
-  module: registry://ghcr.io/kubewarden/policies/psp-capabilities:v0.1.9
+  module: registry://ghcr.io/kubewarden/policies/psp-capabilities:v0.1.8
```

Comment:
The `capabilities-psp` policy version `v0.1.8` is already using the latest name pattern. So you can follow the same changes you did for version `v0.1.9`. 

https://github.com/kubewarden/capabilities-psp-policy/pkgs/container/policies%2Fcapabilities-psp

Reply:
woops, fixed by rebasing https://github.com/kubewarden/kubewarden-controller/pull/254/commits/26b65ad261f08b6999c9f095a85589d538e9afbe! thanks!

Also, bumped and fixed golangci-lint warning.

Reply:
Thanks you!